community/SIG-GOVERNANCE.md

```diff
@@ -19,3 +19,19 @@ SIGs are held accountable to the Maintainers.
 * Meetings should be open to the community unless there is a specific reason for keeping it private.
 * Meeting recordings and/or minutes should be published within 24 hours of the meeting.
 For embargoed decisions, the decisions must be made public as soon as the embargo is lifted.
+
+### Creation
+
+* A new SIG could be in the formation phase by sharing the proposal with the maintainer meeting and receiving unanimous agreement in the maintainer meeting.
+* A new SIG formation phrase should last for more than 3 months and also meet the following milestones:
+
+  * The formation SIG has a clear and actionable goal defined
+  * The formation SIG has interested members from more than 3 organizations
+  * The formation SIG has conducted several (3+) routine meetings to make substantial progress in its agenda
+* After the formation phase, the formation SIG team should present to the maintainer meeting about the current progress. 
+* The unanimous agreement among maintainers could turn the formation SIG into the official SIG. With any disagreement, the proposed SIG will remain in the formation phase.
+
+### Deprecation
+
+* The maintainer meeting should review SIGs each quarter for their activities in the maintainer meeting.
+* The maintainer meeting should deprecate SIGs with no substantial activity in the past 6 months. The unanimous agreement among maintainers could reinstall a deprecated SIG.
```

Comment:
As mentioned in the maintainer meeting today, a vote in that meeting should be enough. 
I think "should" here is good enough, leaves discretion to the maintainers, since we dont have to deprecate after 6 mo.

and +1 on the reinstatement clause.